src/main/kotlin/com/github/ashleybudtke/happyplants/controllers/PlantController.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.ashleybudtke.happyplants.controllers
+
+import com.github.ashleybudtke.happyplants.domain.Plant
+import com.github.ashleybudtke.happyplants.domain.PlantService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class PlantController {
+
+    @Autowired
+    lateinit var plantService: PlantService
+
+    @GetMapping(value = ["/api/plants"])
```

Comment:
Consider not hardcoding and moving to some kind of "paths" file, e.g. 
```suggestion
    @GetMapping(value = [PlantsPaths.BasePath])
```